src/request.rs

```diff
@@ -42,6 +43,15 @@ pub struct AppendOperation {
     requester: Requester,
 }
 
+/// Wrapper message that contains the request, message ID, and the requester ID,
+/// which should also contain the signature(s) if it's sent by the owner(s).
+#[derive(Hash, Eq, PartialEq, PartialOrd, Ord, Clone, Serialize, Deserialize)]
+pub struct Message {
```

Comment:
Should we try and convey that this is a wrapper for Client requests?  Not sure of the best name to do that, but `Message` implies requests and responses to me.